I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 195, in script
    movie play "images/opening.webm"
Exception: Sayer 'movie' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 195, in script
    movie play "images/opening.webm"
  File "C:\Users\100ga\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 890, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\100ga\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 783, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'movie' is not defined.

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
Concert of Liberation DEMO 1.0
Wed Dec 13 00:45:41 2023
